android/src/main/java/com/onegini/mobile/sdk/reactnative/RNOneginiSdk.kt

```diff
@@ -260,7 +266,8 @@ class RNOneginiSdk(reactContext: ReactApplicationContext) : ReactContextBaseJava
 
     @ReactMethod
     @Throws(Exception::class)
-    override fun submitPinAction(flowString: String?, action: String, pin: String?) {
+    fun submitPinAction(flowString: String?, action: String, pin: String?) {
+        //TODO: Add the nullchecks here and make params nullable
```

Comment:
and missed that one too

Reply:
This function is not promise based yet, so we can't really add the nullchecks here.